supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -205,7 +205,11 @@ When combined with privileges, permissions define a user's or resource's overall
 [discrete]
 [[pid]]
 ==== image:images/yes.png[yes] PID (noun)
+<<<<<<< HEAD
 *Description*: In Red Hat Fuse, the persistent identifier (_PID_) of a registered OSGi service is used to identify the service across container restarts. In Red Hat Fuse (Karaf), PIDs map to `.cfg` configuration files located in the `FUSE_HOME/etc/` directory. A `.cfg` file contains a list of attribute-value pairs that configure a service. You can edit any `.cfg` file to configure or reconfigure the corresponding OSGi service.
+=======
+*Description*: In Red Hat Fuse, the _persistent identifier_ (PID) of a registered OSGi service is used to identify the service across container restarts. In Fuse (Karaf), PIDs map to `.cfg` configuration files located in the `FUSE_HOME/etc/` directory. A `.cfg` file contains a list of attribute/value pairs that configure a service. You can edit any `.cfg` file to configure/reconfigure the corresponding OSGi service.
+>>>>>>> e5fa1ab (Apply correct use of italics and quotes to j-r glossary)
```

Comment:
Leftover text from resolving a merge conflict. Should be removed.

Reply:
Well spotted. Must enhance my VSC layout, as I clearly missed this!